app/jobs/course/duplication_job.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+class Course::DuplicationJob < ApplicationJob
+  include TrackableJob
+  include Rails.application.routes.url_helpers
+  queue_as :lowest
+
+  protected
+
+  # Performs the duplication job.
+  #
+  # @param [Course] current_course The course to duplicate.
+  # @param [User] current_user The user that initiated the duplication service.
+  # @param [Hash] duplication_params A hash of duplication parameters.
+  # @param [Array] all_objects All the objects in the course.
+  # @param [Array] selected_objects The objects to duplicate.
+  def perform_tracked(current_course, current_user, duplication_params = {},
+                      all_objects = [], selected_objects = [])
+    instance = Course.unscoped { current_course.instance }
+    ActsAsTenant.with_tenant(instance) do
```

Comment:
```
ActsAsTenant.without_tenant do
   xxx
end
```

Let's standardize and use without tenant.  There's no request in jobs, hence no tenant.